unity-client/Assets/Rendering/Profiles/RenderProfileWorld.cs

```diff
@@ -14,6 +14,7 @@
 [CreateAssetMenu(menuName = "DCL/Rendering/Create World Profile", fileName = "RenderProfileWorld", order = 0)]
 public class RenderProfileWorld : ScriptableObject
 {
+    [SerializeField] public GameObject loadingBlockerPrefab;
```

Comment:
If you don't need the variable as public:
```suggestion
    [SerializeField] private GameObject loadingBlockerPrefab;
```
If you need the variable as public:
```suggestion
    public GameObject loadingBlockerPrefab;
```

Reply:
Solved making the field public as we need it for initializing the blockers pool.